api/api.go

```diff
@@ -234,6 +294,32 @@ func (api *ImportAPI) addDimension(w http.ResponseWriter, r *http.Request) {
 	log.Info("dimension added", log.Data{"instance_id": instanceID, "name": dimensionName, "value": dimensionValue})
 }
 
+func (api *ImportAPI) getDimensionValues(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	instanceID := vars["instanceId"]
+	dimensionName := vars["dimension_name"]
+	dimensionValues, err := api.dataStore.GetDimensionValues(instanceID, dimensionName)
+	if err != nil {
```

Comment:
Should we have another HTTP response code if the instance / dimension is not found?